### ISSUE TITLE
Lighthouse score optimizations

### ISSUE THREAD

**Issue by helms-charity** [hlxsites/merative#238]
Tests were done on these pages:
https://pagespeed.web.dev/analysis/https-qa-merative-com-company/nq8v9d83w2?form_factor=desktop
https://pagespeed.web.dev/analysis/https-qa-merative-com-events-2023-zelta-user-conference/nxyp4g3reu?form_factor=mobile
https://pagespeed.web.dev/analysis/https-qa-merative-com-documents-case-study-uva-case-study/35mgrdvztu?form_factor=desktop
Items called out by Lighthouse:

- all images should have alt text
- all images that are linked, that don't have linked text or an alt tag, should have aria-label added to describe the function of the link
- Check for links that are not crawlable, such as Close:` <a class="close" aria-label="Close">`
- Interactive controls are keyboard focusable

Related tix: https://github.com/hlxsites/merative/issues/247 , https://github.com/hlxsites/merative/issues/248

**Comment by amol-anand** [comment]
It's not a go-live blocker for the pages for phase 2, but we want to get to this as soon as possible which is why I changed this to Phase 2. Internally we are now trying to achieve all 4 100s rather than just performance 100 as part of our VIP program.
